docs/concepts/Security-Best-Practices.md

```diff
@@ -0,0 +1,166 @@
+
+For more information to secure your package feeds, see [3 Ways to Mitigate Risk When Using Private Package Feeds](https://azure.microsoft.com/en-us/resources/3-ways-to-mitigate-risk-using-private-package-feeds/).
+
+### Client trust policies
+
+There are policies that you can opt-into in which you require the packages you use to be signed. This allows you to trust a package author, as long as it is author signed, or trust a package if it is owned by a specific user or account that is repository signed by NuGet.org.
+
+To configure client trust policies, [see the following documentation](../consume-packages/installing-signed-packages.md).
+
+### Lock files
+
+Lock files store the hash of your package’s content. If the content hash of a package you want to install matches with the lock file, it will ensure package repeatability.
+
+To enable lock files, [see the following documentation](../consume-packages/package-references-in-project-files#locking-dependencies).
+
+## Monitor your supply chain
+
+### Publish to NuGet.org
+
+NuGet.org serves as a central repository to over 200,000 unique packages. Whenever you publish a package, NuGet.org will go through numerous validations and indexing that can benefit you in the long term. These can include scanning the package for viruses, [providing a repository signature](../reference/signed-packages-reference.md) on the package, and even protecting your package ID so only you can push updates to it.
+
+To learn more about the benefits of publishing on NuGet.org, see [Package validation and indexing](../nuget-org/publish-a-package.md#package-validation-and-indexing).
+
+### GitHub secret scanning
+
+GitHub scans repositories for NuGet API keys to prevent fraudulent uses of secrets that were accidentally committed. 
+
+To learn more about secret scanning, see [About secret scanning](https://docs.github.com/en/github/administering-a-repository/about-secret-scanning).
+
+### Author Package Signing
+
+[Author signing](../reference/signed-packages-reference.md) allows a package author to stamp their identity on a package and for a consumer to verify it came from you. This protects you against content tampering and serves as a single source of truth about the origin of the package and the package authenticity. 
+
+To author sign a package, see [Sign a package](../create-packages/sign-a-package.md).
+
+### Two-Factor Authentication (2FA)
+
+Enabling two-factor authentication (2FA) can add an extra layer of security when [logging into your GitHub account](https://docs.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa) or the [NuGet.org public package repository](../nuget-org/individual-accounts.md#enable-two-factor-authentication-2fa). It is recommended that you enable two-factor authentication to protect your account.
+
+### Package ID prefix reservation 
```

Comment:
We need to have more here on rationale behind why folks should consider reserving prefixes when they publish their packages to nuget.org as well as when their packages are internal i.e. they should namespace their internal packages and reserve them on nuget.org to prevent substitution.

Reply:
Perhaps namespaces should be added as it's own section here? 

The reason there is no detailed rationale here is that the primary doc should serve as the introduction to the benefits and challenges. This doc is merely telling you that you should consider it & read more about it.

Joel recently updated https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation to clear up some things too FWIW.